public/Day 05/script.js

```
@@ -1,232 +1,154 @@
-
-    /* ---------- THEME SWITCH ---------- */
-    const themeSwitcher = document.getElementById('theme-switcher');
-    
-    if(!document.body.dataset.theme) {
-        document.body.dataset.theme = 'light'; 
+  const calc = new Calculator(
+    document.getElementById('previous-operand'),
+    document.getElementById('current-operand')
+  );
+
+  document.querySelectorAll('.number').forEach(btn =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)